stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/global/internal/GlobalMutableState.kt

```diff
@@ -156,20 +156,11 @@ public class GlobalMutableState private constructor(
          * Gets the singleton of [GlobalMutableState] or creates it in the first call.
          */
         @InternalStreamChatApi
-        public fun create(clientState: ClientState): GlobalMutableState {
+        public fun get(clientState: ClientState): GlobalMutableState {
             return instance ?: GlobalMutableState(clientState).also { globalState ->
                 instance = globalState
             }
         }
```

Comment:
Should we add `synchronized` here to ensure we won't end up having multiple instances in a multi-threading environment?

Reply:
Like it's done here:
https://github.com/android/architecture-components-samples/blob/master/BasicRxJavaSampleKotlin/app/src/main/java/com/example/android/observability/persistence/UsersDatabase.kt

Reply:
I think it is a good idea.
Maybe our instance is not created multiple times from different threads but would prevent use to other race-conditions in the future